backend/src/main/kotlin/hu/bme/sch/cmsch/component/app/ExportAdminController.kt

```diff
@@ -61,14 +63,21 @@ class ExportAdminController(
     }
 
     @ResponseBody
-    @GetMapping("/properties", produces = [ MediaType.APPLICATION_OCTET_STREAM_VALUE ])
-    fun export(auth: Authentication, response: HttpServletResponse): ByteArray {
+    @GetMapping("/properties")
+    fun export(
+        auth: Authentication,
+        response: HttpServletResponse,
+        @RequestParam(required = false, defaultValue = "false") useIsoEncoding: Boolean
+    ): ByteArray {
         val user = auth.getUser()
         if (!permissionControl.validate(user)) {
             throw IllegalStateException("Insufficient permissions")
         }
         response.setHeader("Content-Disposition", "attachment; filename=\"application-live-${clock.getTime()}.properties\"")
-        return generateProperties().toByteArray()
+
+        val charset = if (useIsoEncoding) StandardCharsets.ISO_8859_1 else StandardCharsets.UTF_8
```

Comment:
Should I use `Charset.defaultCharset()` instead of UTF_8?

Reply:
No, UTF-8 will be better. Let's say if we want to export from a local (let's say windows) instance and import it into a linux based instance it would be bad if we had different encodings.